### DISCUSSION TITLE
Set-PnPListItemPermission - The 'name' argument is invalid

### DISCUSSION THREAD

**Discussion by prossum** [pnp/powershell#1175] [Q&A]
Hi, I try to set unique permission on a list item for the owner group.

```
$lib = "MyLib"
$itemId = <itemid>
$ownerGroup = Get-PnPGroup -AssociatedOwnerGroup
$ownerRole = Get-PnPRoleDefinition | Where-Object {$_.RoleTypeKind -eq "Administrator"}
Set-PnPListItemPermissions -List $lib -Identity $itemId -Group $ownerGroup -AddRole $ownerRole -ClearExisting -SystemUpdate
```

But I get an error:
`The 'name' argument is invalid.`

Does anyone have an idea what is wrong?

**Comment by KoenZomers** [comment]
I can reproduce this issue. Created an issue for it at https://github.com/pnp/powershell/issues/1196. Let's continue tracking and discussing this topic there. Closing this discussion.
